graph/src/data/subgraph/mod.rs

```diff
@@ -814,6 +844,15 @@ impl<C: Blockchain> UnvalidatedSubgraphManifest<C> {
             errors.push(SubgraphManifestValidationError::DataSourceBlockHandlerLimitExceeded)
         }
 
+        // For API versions newer than 0.0.5, validate that all mappings uses the same api_version
+        let mappings = self.0.mappings();
+        let versions_iterator = mappings.iter().map(|mapping| &mapping.api_version);
+        if let Err(different_api_versions) =
+            UnifiedMappingApiVersion::try_from_versions(versions_iterator)
```

Comment:
Last comment, couldn't you use `if let Err(different_api_versions) = self.0.unified_mapping_api_version()` here?

Reply:
I think I can. 
I'll try and use it if it works =)

Reply:
It worked! :tada: